lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java

```diff
@@ -88,7 +91,10 @@ public boolean initModules() {
                                                  this.lightyController.getServices());
                 startAndWaitLightyModule(this.openApi);
             }
-
+            callhomePlugin = new NetconfCallhomePluginBuilder(lightyController.getServices(),
+                rncModuleConfig.getNetconfConfig(),
+                rncModuleConfig.getRestconfConfig().getInetAddress().getHostAddress(), 4334).build();
```

Comment:
Introduce configuration for the port

Reply:
In addition, see that this port value is not being used by `NetconfCallhomePlugin constructor` invoked by `NetconfCallhomePluginBuilder#build` where we have just some hardcoded configuration.